tests/buildTools/src/test/scala/tests/ScipBuildToolSuite.scala

```diff
@@ -57,4 +100,22 @@ class ScipBuildToolSuite extends BaseBuildToolSuite {
             |""".stripMargin
     )
   }
+
+  private def hasMention(name: String) =
+    (out: String) => {
+      assert(out.contains(name), s"Expected output to mention [$name]")
```

Comment:
FYI, `clue(..)` can be useful to provide helpful assertion failures
```suggestion
      assert(clue(out).contains(clue(name)), s"Expected output to mention [$name]")
```